plugins/star-rating/frontend/public/templates/feedback-popup.html

```diff
@@ -1,291 +1,408 @@
+                                        email: eventObject.segmentation.email,
+                                        comment: eventObject.segmentation.comment
+                                    },
+                                    timestamp: Date.now(),
+                                    hour: h,
+                                    dow: dow
+                                }])) + '&app_key=' + Countly.app_key + '&device_id=' + device_id + 'sdk_name=javascript_native_web&sdk_version=' + sdk_version + '&timestamp=' + Date.now() + '&hour=' + h + '&dow=' + dow,
+                                "success": function(response) {
+                                },
+                                "error": function() {
+                                }
+                            });
+                            document.getElementsByClassName("success-modal-content")[0].style.display = "block";
+                            document.getElementsByClassName("modal-content")[0].style.display = "none";
+                        } else {
+                            addClass(document.getElementById('contact-me-email'), 'countly-feedback-verification-fail');
+                        }
+                    } else {
+                        eventObject["segmentation"].comment = document.getElementById('countly-feedback-comment-textarea').value;
+                        if (document.getElementById('contact-me-email').value.length > 0) {
+                            eventObject["segmentation"].email = document.getElementById('contact-me-email').value;
+                            eventObject["segmentation"].contactMe = true;
+                        }
+                        $.ajax({
+                            "type": "GET",
+                            "url": Countly.url + '/i?events=' + encodeURIComponent(JSON.stringify([{
+                                key: "[CLY]_star_rating",
+                                count: 1,
+                                segmentation: {
+                                    widget_id: widget_id,
+                                    contactMe: eventObject.segmentation.contactMe,
+                                    platform: "Web",
+                                    app_version: 1,
+                                    platform_version_rate: "",
+                                    rating: eventObject.segmentation.rating,
+                                    email: eventObject.segmentation.email,
+                                    comment: eventObject.segmentation.comment
+                                },
+                                timestamp: Date.now(),
+                                hour: h,
+                                dow: dow
+                            }])) + '&app_key=' + Countly.app_key + '&device_id=' + device_id + 'sdk_name=javascript_native_web&sdk_version=' + sdk_version + '&timestamp=' + Date.now() + '&hour=' + h + '&dow=' + dow,
```

Comment:
Same here. Of course question arises why we have two of the calls, etc, which leads to a bad code structuring. But ok :D